Bridge and network cleanup (#267)
Consensys/doc.zk-evm: docs/use-mainnet/bridges-of-linea/how-to-bridge-erc20-tokens.mdx

---
title: How do I bridge ERC20 tokens?
sidebar_position: 2
---

:::note

We recommend that only tech operators that are providing liquidity use our bridge for ERC20 tokens. For everyone else, please take a look at our [ecosystem portal](https://linea.build/apps) and filter for bridges.

> **ERC20 tokens can only be bridged via manual claiming, which requires you to cover the ETH fee on the destination layer. Make sure your wallet is properly funded before beginning this process! Checkout the [tutorial](/use-mainnet/bridges-of-linea/how-to-bridge-eth) on how to bridge ETH to learn how to use our bridge and appropriately fund your wallet.**

Reviewer: ```suggestion
**ERC20 tokens can only be bridged via manual claiming, which requires you to cover the ETH fee on the destination layer. Make sure your wallet is properly funded before beginning this process! Check out the [tutorial](/use-mainnet/bridges-of-linea/how-to-bridge-eth) on how to bridge ETH to learn how to use our bridge and appropriately fund your wallet.**
```

:::

1. Go to our token-bridge [here](https://bridge.linea.build/).

> 2. Connect your wallet in top right corner of the page.

Reviewer: ```suggestion
2. Connect your wallet in the top-right corner of the page.
```

> 3. On the [token-bridge](https://bridge.linea.build/) go to the token selection menu by clicking on the current token symbol (ETH in this case).

Reviewer: ```suggestion
3. On the [token bridge](https://bridge.linea.build/), go to the token selection menu by clicking on the current token symbol (ETH in this case).
```
